src/components/common/PeopleAndRoles.vue

```diff
@@ -55,12 +55,24 @@
             <v-icon v-else>mdi-circle-small</v-icon>
             <span class="rule-item-txt">{{rule.text}}</span>
           </li>
+          <li v-if="rule.id === RuleIds.NUM_APPLICANT_PERSON" :key="index">
+            <v-icon v-if="validApplicantPerson" color="green darken-2" class="dir-valid">mdi-check</v-icon>
+            <v-icon v-else-if="getShowErrors" color="error" class="dir-invalid">mdi-close</v-icon>
+            <v-icon v-else>mdi-circle-small</v-icon>
+            <span class="rule-item-txt">{{rule.text}}</span>
+          </li>
+          <li v-if="rule.id === RuleIds.NUM_APPLICANT_ORG" :key="index">
+            <v-icon v-if="validApplicantOrg" color="green darken-2" class="dir-valid">mdi-check</v-icon>
+            <v-icon v-else-if="getShowErrors" color="error" class="dir-invalid">mdi-close</v-icon>
+            <v-icon v-else>mdi-circle-small</v-icon>
+            <span class="rule-item-txt">{{rule.text}}</span>
+          </li>
         </template>
       </ul>
     </section>
 
     <!-- Start by Adding the Completing Party -->
-    <div v-if="orgPersonList.length === 0">
+    <div v-if="(orgPersonList.length === 0) && requireCompletingParty">
```

Comment:
All other filings require a completing party, but not restoration, so I implemented this local getter (which is declared in mixin).